source/_static/js/pid-tune.js

```diff
@@ -0,0 +1,216 @@
+class Sample{constructor(time_in,value_in){this.time=time_in;this.value=value_in;}}

```

Comment:
Why is this touched?

Reply:
Should not have been, will have to go look at the history to see how this ended up getting committed.